src/emcpy/stats.py

```diff
@@ -193,3 +195,31 @@ def get_weighted_mean(data, weights, axis=None):
         'data and weights mis-match array size')
 
     return _np.average(data, weights=weights, axis=axis)
+
+
+def get_linear_regression(x, y):
+    """
+    Calculate linear regression between two sets of data.
+    Fits a linear model with coefficients to minumize the
+    residual sum of squares between the observed targets
+    in the dataset, and the targets predicted by the linear
+    approximation.
+    INPUT
```

Comment:
Parameters, Return now, right?

Reply:
Yes.

Reply:
Once the doc string for the linear regression is fixed to match the new format, I'm happy with merging.